chart/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    apps.kubernetes.io/component: controller
+    apps.kubernetes.io/name: terraform-controller
+  name: terraform-controller
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Cluster admin really needed?

Reply:
This I plan to improve next, I'll keep executor SA as is though.

Reply:
@donovanmuller https://github.com/AbsaOSS/terraform-controller/issues/20 created to track this